### ISSUE TITLE
Setting the namespace 

### ISSUE THREAD

**Issue by cnkygmr** [GigaDroid/flutter_udid#47]
Incorrect package="de.gigadroid.flutter_udid" found in source AndroidManifest.xml: /Users/xxx/.pub-cache/hosted/pub.dev/flutter_udid-2.1.0/android/src/main/AndroidManifest.xml.
Setting the namespace via the package attribute in the source AndroidManifest.xml is no longer supported.

**Comment by GigaDroid** [comment]
Hey, could you please provide more information? Does the build fail completely? What flutter version are you using? Please post the full build log as well.

**Comment by cnkygmr** [comment]
so you need remove AndroidManifest.xml package attribute and you need android build gradle to like that

```
android {
    if (project.android.hasProperty('namespace')) {
        namespace 'de.gigadroid.flutterudid'
    }
}
```

**Comment by GigaDroid** [comment]
Just released 2.1.1 to address this
